cmd/tools/studio/cfg/copyenv.go

```diff
@@ -0,0 +1,85 @@
+package cfg
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var heyCMD = &cobra.Command{
+	Use:   "copyenv",
+	Short: "Copy env variables configmap for a microservice in one environment to another environment",
+	Long: `
+	Copy env variables configmap for a microservice in one environment to another environment.
+
+	go run main.go tools studio cfg copyenv --application cde2e951-d40a-3548-8b45-64c0ded97940 \
+		--microservice-name frontend \
+		--from-env test \
+		--to-env prod
+	`,
+	Run: func(cmd *cobra.Command, args []string) {
+		logrus.SetFormatter(&logrus.JSONFormatter{})
+		logrus.SetOutput(os.Stdout)
+		logContext := logrus.StandardLogger()
+
+		sourceMicroserviceName := viper.GetString("tools.studio.cfg.microservice-name")
+		sourceEnvironment := viper.GetString("tools.studio.cfg.from-env")
+		destinationEnvironment := viper.GetString("tools.studio.cfg.to-env")
+		application := viper.GetString("tools.studio.cfg.application")
+
+		sourceConfigMapName := fmt.Sprintf("%s-%s-env-variables", sourceEnvironment, sourceMicroserviceName)
+		namespace := fmt.Sprintf("application-%s", application)
+		logContext.WithFields(logrus.Fields{
+			"k8sNamespace":     namespace,
+			"k8sConfigmap":     sourceConfigMapName,
+			"microserviceName": sourceMicroserviceName,
+		}).Info("Begins to copy env var configmap")
```

Comment:
```suggestion
		logContext = logContext.WithFields(logrus.Fields{
			"k8sNamespace":     namespace,
			"k8sConfigmap":     sourceConfigMapName,
			"microserviceName": sourceMicroserviceName,
		})
		logContext.Info("Begins to copy env var configmap")
```

This way you save the properties that are setup into the `logContext` for later use.

Reply:
Done